test/integration/api/projects.js

```diff
@@ -69,7 +69,7 @@ describe('api: /projects', () => {
               .set('X-Extended-Metadata', 'true')
               .expect(200)
               .then(({ body }) => {
-                body.verbs.length.should.be.greaterThan(39);
+                body.verbs.length.should.be.greaterThan(43);
```

Comment:
Should be 41?

Reply:
This is such an awkward test because it doesn't change when new verbs are added. I think the number was not incremented last time? I ran it and it was expecting 44 verbs.

Reply:
Oh very interesting! Would it be worth changing it to an equality check rather than `greaterThan` so that we remember  to update it more frequently?